nuget/nunitlite/nunitlite.nuspec

```diff
@@ -6,7 +6,8 @@
     <version>$version$</version>
     <authors>Charlie Poole, Rob Prouse</authors>
     <owners>Charlie Poole, Rob Prouse</owners>
-    <license type="file">LICENSE.txt</license>
+    <license type="expression">MIT</license>
+    <licenseUrl>https://licenses.nuget.org/MIT</licenseUrl>
```

Comment:
Is this url correct?  Without it, the license points to this URL anyway, see NUnit3TestAdapter, and it then doesn't have the fields filled out.   Do you have an another package as an example?